### ISSUE TITLE
Medium-blue buttons have insufficiently high contrast with white text

### ISSUE THREAD

**Issue by neel-a-mehta** [seattle-uat/civiform#1476]
We have several buttons on CiviForm that are a medium blue, with white text. The trouble is that the contrast ratio for these buttons isn't high enough: 3.7, instead of the recommended 4.5+.

![image](https://user-images.githubusercontent.com/77128518/123185605-28f96200-d44b-11eb-95ce-2ed9bc750a02.png)

Instead, use a darker color for buttons throughout the app. [Use this contrast checker](https://webaim.org/resources/contrastchecker/) as needed.

**Comment by baipai** [comment]
I think this one may have already been fixed? I think the light blue one is just for the temporary setting? The dark blue color right now is 8.44:1.

<img width="1351" alt="Screen Shot 2021-06-24 at 9 11 24 AM" src="https://user-images.githubusercontent.com/11348330/123187300-7a960100-d4cc-11eb-9870-15ce1b6c298d.png">

**Comment by neel-a-mehta** [comment]
Yes, good point -- though we should see if those medium-blue buttons appear anywhere else in the app. If so, we should replace them with the darker blue buttons.
